tangent_systems_invoice_creation/models/project_task.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models,_
+from odoo.odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    invoice_number = fields.Many2one(comodel_name="account.move", ondelete='restrict')
```

Comment:
relation field should always end with an id ex. invoice_id instead of invoice_number